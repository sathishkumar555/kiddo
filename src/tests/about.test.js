import React from 'react';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import Header from '../components/About';

test('should render About correctly', () => {
  const renderer = new ReactShallowRenderer();
  renderer.render(<Header />);
  expect(renderer.getRenderOutput()).toMatchSnapshot();
  console.log(renderer.getRenderOutput());
});

