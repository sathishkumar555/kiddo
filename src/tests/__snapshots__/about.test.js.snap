// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render About correctly 1`] = `
<Unknown>
  <div>
    This is the test environment
  </div>
</Unknown>
`;
